routes/basic.rb

```diff
@@ -84,6 +87,9 @@
 get '/reset' do
   redirect '/reports/list' unless valid_session?
 
+  @admin = true if is_administrator?
```

Comment:
Users should be able to access the pw reset functionality. Also, the plugin line should be removed.

Reply:
Bah, this is for the template. Ignore this and previous comment. Sorry about that.